### PR TITLE
Add support for openresty/openresty

### PR DESCRIPTION
**Pull request by krishvoor**
This PR adds build-scripts for the last release/tag version per the community.

What is OpenResty?
OpenResty is a fledged web platform that integrates the standard Nginx core and LuaJIT.

Signed-off-by: Krishna Harsha Voora <krishvoor@in.ibm.com>

### CODE REVIEW

File: o/openresty/redhat_ubi8/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+        Examples:
+                ==========
+                Build Only
+                ==========
+                # by default, build "lasttestedrelease" which is v1.19.3.1  marked as a 'Tagged'  as on 7 Nov 2020
+                # both the commands below build the v1.19.3.1: branch
+                $ docker run --rm -v `pwd`:/ws openresty_ibm bash -l -c "cd /ws; ./build.sh 2>&1 | tee output.log"
+                $ docker run --rm -v `pwd`:/ws openresty_ibm bash -l -c "cd /ws; ./build.sh lasttestedrelease 2>&1 | tee output.log"
+
```
Comment (Abhijit-Mane) [inline o/openresty/redhat_ubi8/README.md:20]: Maybe good to add a build argument to trigger tests if desired.

**Review by Abhijit-Mane — APPROVED** [review]
minor suggestions, otherwise lgtm.

File: o/openresty/redhat_ubi8/build.sh [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+cd ..
+export PATH=$OPENRESTY_PREFIX/bin:$OPENRESTY_PREFIX/nginx/sbin:$PATH
+echo "########### Nginx Version"
+nginx -V
+ldd `which nginx`|grep -E 'luajit|ssl|pcre'
+cd /opt/openrest/
+tar cf $HOME/openrest.tar /opt/openrest/
+cp $HOME/openrest.tar  /ws/
+
+
```
[the remark this answers is not in the document]
Comment (krishvoor) [reply o/openresty/redhat_ubi8/build.sh:116]: Yes, we can put a placeholder for the tests.

File: o/openresty/redhat_ubi8/README.md (hunk 1 above)
Comment (krishvoor) [reply o/openresty/redhat_ubi8/README.md:20]: Sure, will add.

**Pull request closed without merging**